.gitignore

```diff
@@ -0,0 +1,160 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
```

Comment:
I'd rather not include such an extensive gitignore instead of the minimal amount that's needed.